Converters/Rhino/Speckle.Converters.RhinoShared/Extensions/GeometryBaseExtensions.cs

```diff
@@ -0,0 +1,23 @@
+namespace Speckle.Converters.Rhino.Extensions;
+
+public static class GeometryBaseExtensions
+{
+  /// <summary>
+  /// Getting translation vector from origin to the Geometry bbox Center (if geometry is far from origin and translation needed)
+  /// </summary>
+  /// <returns>
+  /// Vector from origin to Geometry bbox center (if translation needed), otherwise zero-length vector.
+  /// </returns>
```

Comment:
Oh forgot, can you add this in-code comment link to documentation: https://wiki.mcneel.com/rhino/farfromorigin

Reply:
(and explain this issue is caused by rhino api geometry operations being double-precision, but rhino mesh is single-precision)